packages/core/src/registries/NativeStyleRegistry.ts

```diff
@@ -14,9 +15,12 @@ export abstract class NativeStyleRegistry {
    * @returns {void}
    */
   public static define(name: string, css: string): void {
-    if (register.has(name)) {
+    if (register.has(name) && isLocalhost) {
       throw new DuplicateStyleError(name);
+    } else if (register.has(name) && !isLocalhost) {
```

Comment:
There is no need to check `register.has(name)` twice.

Reply:
Fixed